app/ui/ui-tiers-and-addons/src/main/kotlin/com/hedvig/android/tiersandaddons/QuoteCard.kt

```diff
@@ -367,7 +368,13 @@ private fun QuoteCard(
         }
         Column(Modifier.weight(1f)) {
           HorizontalItemsWithMaximumSpaceTaken(
-            startSlot = { HedvigText(displayName) },
+            startSlot = {
+              HedvigText(
+                text = displayName,
+                maxLines = if (quoteCardState.showDetails) Int.MAX_VALUE else 1,
```

Comment:
I played a lot with trying to get a similar animation here as https://file.notion.so/f/f/db6cdb10-2d5c-4997-afa5-6f7e1f8f9a9b/6b20f0c2-b209-40fe-8cc4-32500ffdc4c7/RPReplay_Final1737972149.mp4?table=block&id=188c3f71-cb0a-80c1-a512-e4da1a979eec&spaceId=db6cdb10-2d5c-4997-afa5-6f7e1f8f9a9b&expirationTimestamp=1738176202882&signature=xT5iZyVflh-souI3q-f6hEJ9KeXxPPj9lDQal0JMJ8Q&downloadName=RPReplay_Final1737972149.MP4 but I think we just can't do it reasonably as we don't get this as a built-in animation

Reply:
The no animation choice I made here looked better than adding AnimatedContent, shared elements etc. It's just simple and it looks okay this way.